tests/API/XCCDF/guide/benchmark_id_test-ds.xml

```diff
@@ -0,0 +1,185 @@
+      <cpe-item name="cpe:/o:redhat:enterprise_linux:7">
+            <title xml:lang="en-us">Red Hat Enterprise Linux 7</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="ssg-rhel7-cpe-oval.xml">oval:ssg-installed_OS_is_rhel7:def:1</check>
+      </cpe-item>
+      <cpe-item name="cpe:/o:redhat:enterprise_linux:7::server">
+            <title xml:lang="en-us">Red Hat Enterprise Linux 7 Server</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="ssg-rhel7-cpe-oval.xml">oval:ssg-installed_OS_is_rhel7:def:1</check>
+      </cpe-item>
+      <cpe-item name="cpe:/o:redhat:enterprise_linux:7::client">
+            <title xml:lang="en-us">Red Hat Enterprise Linux 7 Client</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="ssg-rhel7-cpe-oval.xml">oval:ssg-installed_OS_is_rhel7:def:1</check>
+      </cpe-item>
+      <cpe-item name="cpe:/o:redhat:enterprise_linux:7::computenode">
+            <title xml:lang="en-us">Red Hat Enterprise Linux 7 ComputeNode</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="ssg-rhel7-cpe-oval.xml">oval:ssg-installed_OS_is_rhel7:def:1</check>
+      </cpe-item>
+      <cpe-item name="cpe:/o:redhat:enterprise_linux:7::workstation">
+            <title xml:lang="en-us">Red Hat Enterprise Linux 7 Workstation</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="ssg-rhel7-cpe-oval.xml">oval:ssg-installed_OS_is_rhel7:def:1</check>
+      </cpe-item>
+      <cpe-item name="cpe:/o:centos:centos:7">
+            <title xml:lang="en-us">CentOS 7</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="ssg-rhel7-cpe-oval.xml">oval:ssg-installed_OS_is_centos7:def:1</check>
+      </cpe-item>
+      <cpe-item name="cpe:/o:scientificlinux:scientificlinux:7">
+            <title xml:lang="en-us">Scientific Linux 7</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="ssg-rhel7-cpe-oval.xml">oval:ssg-installed_OS_is_sl7:def:1</check>
+      </cpe-item>
+    </cpe-list>
+  </ds:component>
```

Comment:
The CPE component is not used, can you please remove it? It's not necessary for testing purposes.